cmd/manager/main.go

```diff
@@ -158,6 +165,12 @@ func main() {
 	// work
 	stopCh := signals.SetupSignalHandler()
 
+	accountInitClient, err := client.New(cfg, client.Options{})
```

Comment:
Could this use the same client as the account watcher? Not sure if it matters but just for less memory usgae

Reply:
Yeah, I was wondering what the best play would be here.  I have no problem doing that, but I'd want to rename it to be a generic client that we pass into any other startup process there.

Reply:
Created a generic client.